### PR TITLE
Add flag to disable TLS for development

### PR DESCRIPTION
**Pull request by loganmac**

### CODE REVIEW

File: cmd/launcher/options.go [hunk 1]
```diff
@@ -144,6 +145,11 @@ func parseOptions() (*options, error) {
 			env.Bool("KOLIDE_LAUNCHER_DEBUG", false),
 			"Whether or not debug logging is enabled (default: false)",
 		)
+		flDisableTLS = flag.Bool(
+			"disable_tls",
+			env.Bool("KOLIDE_LAUNCHER_DISABLE_TLS", false),
+			"Disable TLS encryption",
```
Comment (groob) [inline cmd/launcher/options.go]: can this be more specific, specify that it's for control?

File: cmd/launcher/launcher.go [hunk 2]
```diff
@@ -381,6 +381,9 @@ func main() {
 		if opts.insecureTLS {
 			controlOpts = append(controlOpts, control.WithInsecureSkipVerify())
 		}
+		if opts.disableControlTLS {
+			controlOpts = append(controlOpts, control.WithDisableTLS())
```
Comment (NickTitle) [inline cmd/launcher/launcher.go]: s/withDisableTLS/withDisableControlTLS

Comment (loganmac) [reply cmd/launcher/launcher.go]: That's in the `control` package. Thoughts to keep it as is?

Comment (NickTitle) [reply cmd/launcher/launcher.go]: oh now i see, nvm 👍

**Pull request merged**